dev/com.ibm.ws.microprofile.metrics.monitor_fat/fat/src/com/ibm/ws/microprofile/metrics/monitor_fat/MetricsMonitorTest.java

```diff
@@ -260,6 +230,40 @@ public void testEnableDisableFeatures() throws Exception {
       	checkStrings(getHttpsServlet("/metrics"), 
       		new String[] {}, 
       		new String[] { "vendor:" });
+            
+        if (componenttest.topology.impl.JavaInfo.forServer(server).JAVA_VERSION<=8){
```

Comment:
`JAVA_VERSION` just returns `JavaInfo.forCurrentVM().majorVersion();`. I think you should use `JavaInfo.forServer(server).majorVersion()` instead.

Reply:
Change tested and committed.